### ISSUE TITLE
MEGAHIT instead of metaspades?

### ISSUE THREAD

**Issue by cjfields** [abishara/athena_meta#25]
I noticed in the Athena publication that the marine sediment read cloud sample was assembled with MEGAHIT prior to using Athena, but I couldn't find any documentation on how MEGAHIT was run and whether any additional post-processing was needed 

As MEGAHIT is quite a bit faster than MetaSPAdes and uses less memory, is there any reason to preferentially use metaspades (e.g. better assemblies)?  And if we try MEGAHIT, are there any issues we can expect to run into?  Thanks!

**Comment by abishara** [comment]
Hi,

MEGAHIT was run using its default parameters as specified under "Assembly of marine-sediment sample" in the NBT publication.  We then aligned the raw reads back to these assembled contigs, which is required by Athena anyway, and filtered contigs with very low coverage.  This additional post processing step removes seed contigs that won't be further assembled anyway and enables Athena to finish much faster than without it.

The main reason to use metaspades over MEGAHIT is that the quality of the seed contigs (the distribution of their lengths) is better.  Empirically we observed better results from Athena when it is provided with longer input seed contigs.  Of course, it should still work just fine should you choose to use MEGAHIT instead to reduce the total resources used.  For our marine sediment sample in the publication, we had to use MEGAHIT as the sample was unable to be assembled using metaspades.

Hope this helps.

alex
